CDP4ShellDialogs/Views/ModelOpeningDialog.xaml

```diff
@@ -140,22 +141,22 @@
             </dxg:GridControl>
         </dxlc:LayoutGroup>
         <views1:LoadingControl Grid.Row="1" Visibility="{Binding IsBusy, Converter={dx:BooleanToVisibilityConverter}}"/>
-        <StackPanel Grid.Row="2" Margin="5" Visibility="{Binding IsModelScreen, Converter={dx:BooleanToVisibilityConverter}}">
-            <TextBlock TextWrapping="Wrap" Background="Cornsilk"
+        <StackPanel Grid.Row="2" Margin="5,5,5,5" Background="Cornsilk" Visibility="{Binding IsModelScreen, Converter={dx:BooleanToVisibilityConverter}}">
+            <TextBlock TextWrapping="Wrap" Margin="5,5,5,5"
                        Text="Double click to open the active iteration of this Engineering Model"></TextBlock>
         </StackPanel>
         <dxlc:LayoutGroup Grid.Row="3" Height="30" Margin="5">
             <Button Content="Back"
                     Visibility="{Binding Path=IsIterationScreen, Converter={dx:BooleanToVisibilityConverter}}"
                     Margin="5"
                     Width="60"
-                    HorizontalAlignment="Left"
+                    HorizontalAlignment="Right"
                     Command="{Binding BackCommand}"/>
-            <Button Content="Next"
+            <Button Content="Select Iteration"
                     Visibility="{Binding IsModelScreen, Converter={dx:BooleanToVisibilityConverter}}"
                     Margin="5"
-                    Width="60"
-                    HorizontalAlignment="Left"
+                    Width="120"
+                    HorizontalAlignment="Right"
                     Command="{Binding NextCommand}">
```

Comment:
maybe also rename the NextCommand to SelectIterationCommand

Reply:
Changed